03app_labomap/03app_labomap.c

```diff
@@ -0,0 +1,168 @@
+#include <stdio.h>
+#include "nrf52833.h"
+#include "busywait.h"
+
+/*
+sources:
+    https://github.com/DotBots/DotBot-firmware/blob/main/bsp/nrf/i2c.c
```

Comment:
read this comment and remove what;s not accurate. I expect to see:
- a description of what this app does
- how the SHT31 is connected to the nRF52833